recipes/calcam/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "calcam" %}
+{% set version = "2.12.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/calcam-{{ version }}.tar.gz
+  sha256: ec37dae2d7bb9bf4a6a8e110328bcafb6f4dbc8d2906b856daccff04551bf2a3
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  entry_points:
+    - calcam = calcam:start_gui
+  number: 0
+
+requirements:
+  host:
+    - python >=3.5
+    - setuptools >=61.0
+    - pip
+  run:
+    - python >=3.5
+    - matplotlib-base
+    - opencv
+    - pyqt
+    - scipy
+    - vtk >=7,!=9.1.*
+  run_constrained:
+    - libstdcxx-ng >=13
```

Comment:
What is the reason for this entry?

Reply:
This is derived from the original recipe coded in the source, but it might be no problem if removeed.

Reply:
According to @ssilburn,
`vtk` requires `libstdcxx-ng` >=13. Ubuntu + Anaconda's default version (11) causes a crash. Version >=13 works properly.